gapis/capture/capture.go

```diff
@@ -129,14 +129,16 @@ func ResolvePerfettoFromPath(ctx context.Context, p *path.Capture) (*PerfettoCap
 }
 
 // Import imports the capture by name and data, and stores it in the database.
-func Import(ctx context.Context, name string, src Source) (*path.Capture, error) {
+func Import(ctx context.Context, name string, key string, src Source) (*path.Capture, error) {
```

Comment:
Why don't we just use a hash of the capture's data?

Reply:
Hashing can take long time depending on the capture size which can be in GBs.

Reply:
Merging this version and if we decide to use hash we can update it in another PR.